www-servers/nginx/nginx-1.19.9.ebuild

```diff
@@ -278,7 +278,7 @@ done
 # @TODO: jbergstroem: remove on next release series
 IUSE="${IUSE} nginx_modules_http_spdy"
 
-CDEPEND="
+COMMON_DEPEND="
```

Comment:
I wouldn't bother with stuff like this. You are right because we should avoid using XDEPEND where X is a single-character - they're supposed to be reserved for future EAPIs.

But it's not worth messing with existing style and confusing maintainers, etc. If we're going to do it, we should do that in a separate PR and make it really clear in the commit message(s).

Reply:
Good point. By now it had become so habitual that I hadn't thought about adding a note on it